package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "bytes",
   "description": "Utility to parse a string bytes to bytes and vice-versa",
-  "version": "2.4.0",
+  "version": "3.4.0",
```

Comment:
Wouldn't you consider the fact that the API can return different values compared to last time a breaking change though?